src/interfaces/chain.ts

```diff
@@ -147,6 +147,8 @@ export interface Chain {
   supportsGetPublicKeyFromSignature: boolean
   /** Whether the chain supports resources */
   supportsResources: boolean
+  /** Whether the chain suppports ERC712 data type signature*/
```

Comment:
Please tweak the comment - this is generic for all chains - so change to something like:
/** Whether the chain's signMessage feature supports signing a typed data object (ex: ERC712 data type for Ethereum)*/